blocks/log.py

```diff
@@ -19,7 +19,7 @@ class AbstractTrainingStatus(object):
         The number of iterations done.
     epochs_done : int
         The number of epochs done.
-    _epoch_ends : list
+    epoch_ends : list
```

Comment:
Oops, we can not do that right now. Currently private content of the log and the status is distinguished from public by a leading underscore. The `Printing` extension ignores private attributes. This is how I keep the output reasonably compact. While I have some ideas of better solutions (adding a sort of priorities to the attributes), I suggest we keep it like this for the 0.1 version.